### ISSUE TITLE
Electra Style Models

### ISSUE THREAD

**Issue by MdavoodiMS** [NVIDIA/TensorRT#2551]
## Description

Hello,

Is electra style models supported by TensorRT? 

It seems that we need add attention scores to add rel_pos_bias before nn.softmax which isn’t supported.
Also, there is a layer named gate_ur_linear which is different with bert model

![image](https://user-images.githubusercontent.com/120752318/208184065-dc1834d3-688a-4ab9-805e-b2ff28bcd5ad.png)
![image](https://user-images.githubusercontent.com/120752318/208184081-c167a87f-e16d-47d7-8f1b-5d811ddbf1bc.png)

## Environment

**TensorRT Version**: 
**NVIDIA GPU**: 
**NVIDIA Driver Version**: 
**CUDA Version**: 
**CUDNN Version**: 
**Operating System**: 
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**:

**Comment by zerollzeng** [comment]
Can you export the model to onnx? if yes then very likely TRT can support it.

**Comment by MdavoodiMS** [comment]
Yes; We are able to convert the model to ONNX and do inference on it. What step should we take to have it supported by TRT after this conversion?

**Comment by zerollzeng** [comment]
To quick check it, you can use trtexec(`trtexec --onnx=model.onnx`) or onnx-tensorrt.

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, please reopen if you still have question, thanks!
